content/pages/disability-benefits/apply/dependents.md

```diff
@@ -0,0 +1,304 @@
+  - heading: Information for family members and caregivers
+    links:
+    - url: /families-caregivers/
+      title: Family Member and Caregiver Benefits
+      description: Find out which benefits you may qualify for as the family member or caregiver of a Veteran or Servicemember.
+---
+
+<div class="va-introtext">
+
+Find out how to add a dependent spouse, child, or parent to your VA disability compensation benefits so you can get a higher payment to help support your family.
+
+</div>
+
+<div class="feature" markdown="1">
+
+### Can I add a dependent to my VA disability compensation benefits?
+
+You may be able to add a dependent spouse, child, or parent to your benefits if you meet the requirements listed below.
+
+**Both of these must be true. You:**	
+
+- Are receiving VA disability compensation, **and**
+- Have a 30% or higher disability rating <br>
+
+Adding a dependent may make you eligible to receive a higher compensation payment (also called a benefit rate).
+
+<br>
+
+### Who can I add as a dependent?
+
+**You can add your:**
+
+- Spouse (we recognize same-sex and common-law marriages)
+- Unmarried child (including a biological, adopted, or stepchild) who: 
+  - Is under 18 years old, **or**
+  - Is between the ages of 18 and 23 years old and enrolled in school full time, **or**
+  - Became seriously disabled before they turned 18 
+- Parent, if you’re directly caring for them and their income and net worth are below a certain amount
+
+</div>
+
+### When should I add a dependent to my benefits?
+
+This depends on where you are in the process of applying for disability benefits.
+
+#### If you haven’t yet applied for disability benefits
+
+You can apply to add your dependent at the same time you file your disability claim.
+
+If you receive a 30% or higher disability rating, we’ll automatically consider your dependent when we determine your disability compensation.
+
+[Find out how to apply for disability benefits](/disability-benefits/apply/). 
+
+
+#### If you have one or more dependents, but didn’t add them when you filed your original disability claim
+
+If you’ve received a 30% or higher disability rating, you can apply online to add your dependent now.
+
+#### If something changes in your family status after you’ve started receiving compensation
+
+You can apply online to add any new dependents right away. This could affect your dependent’s eligibility for benefits.
+
+**You can add a dependent if:**
+
+- You get married 
+- You have or adopt a child
+- Your child is between 18 and 23 years old and enrolled in school full time
+- Your child becomes severely disabled before turning 18
+- You become the caregiver for a parent whose income and net worth is below a certain amount
+
+<br>
+
+### Ready to apply? 
+
+#### To add a dependent child or spouse
+
+You can apply online. You may also need to provide more information or forms along with your application.
+
+<div id="react-applicationStatus" class="static-page-widget">
```

Comment:
You can remove this div for now, I think. Including this will hide the button in order to hook it into our save in progress widget, which isn't set up for 686 yet, I think.

Reply:
@jbalboni OK, thank you for taking a look at them.